src/Codeception/Util/ReflectionHelper.php

```diff
@@ -163,7 +174,7 @@ public static function getDefaultValue(ReflectionParameter $param)
      *
      * @param mixed $value
      *
-     * @return string
+     * @return string|bool
```

Comment:
Where do you see bool? It retuns strings only.

Reply:
json_encode returns false if it fails.

Reply:
`json_encode` returns `false` if it fails.

Reply:
Maybe we should start using JSON_THROW_ON_ERROR or use var_export instead.